### PR TITLE
docs(export(assets_version=)): Add caveat that not all `assets_version` values may work

### PR DESCRIPTION
**Pull request by schloerke**

### CODE REVIEW

File: R/export.R [hunk 1]
```diff
@@ -13,7 +13,8 @@
 #' @param package_cache Cache downloaded binary WebAssembly packages. Defaults
 #'    to `TRUE`.
 #' @param assets_version The version of the Shinylive assets to use in the
-#'    exported app. Defaults to [assets_version()].
+#'    exported app. Defaults to [assets_version()]. Note, not all custom assets
+#'    versions may work with this release of \pkg{shinylive}.
```
Comment (gadenbuie) [inline R/export.R:17]: What kinds of issues do you have in mind? This is a decent "cover our butts" warning but maybe we could be more specific. Maybe we should also link to the shinylive release page so it's clear what our options are?

Comment (schloerke) [reply R/export.R:17]: Yes, more of a "cover our butts" situation. 

Yes, a link would be good!

Comment (schloerke) [inline R/export.R:17]:
```suggestion
#'    versions may work with this release of \pkg{shinylive}. Please visit 
#'    [`shinylive asset releases`](https://github.com/posit-dev/shinylive/releases)
#'    website to learn more about the available versions.
```

File: R/export.R [hunk 2]
```diff
@@ -13,7 +13,10 @@
 #' @param package_cache Cache downloaded binary WebAssembly packages. Defaults
 #'    to `TRUE`.
 #' @param assets_version The version of the Shinylive assets to use in the
-#'    exported app. Defaults to [assets_version()].
+#'    exported app. Defaults to [assets_version()]. Note, not all custom assets
+#'    versions may work with this release of \pkg{shinylive}. Please visit the
+#'    [`shinylive asset releases`](https://github.com/posit-dev/shinylive/releases)
```
Comment (schloerke) [inline R/export.R:18]:
```suggestion
#'    [shinylive asset releases](https://github.com/posit-dev/shinylive/releases)
```

**Pull request merged**